qa/erasure-code/ec-rados-plugin=jerasure-k=8-m=6-crush.yaml

```diff
@@ -0,0 +1,47 @@
+roles: 
+  - [mon.a, mgr.y, osd.0, osd.4, osd.8, osd.12]
+  - [mon.b, osd.1, osd.5, osd.9, osd.13] 
+  - [mon.c, osd.2, osd.6, osd.10, osd.14] 
+  - [mgr.x, osd.3, osd.7, osd.11, osd.15, client.0]
+tasks:
+- rados:
+    clients: [client.0]
+    ops: 400000
+    max_seconds: 600
+    max_in_flight: 64
+    objects: 1024
+    size: 16384
+    ec_pool: true
+    write_append_excl: false
+    erasure_code_crush:
+      name: jerasure86crush
+      id: 86
+      type: erasure
+      min_size: 3
+      max_size: 6
+      steps: [
+        'set_chooseleaf_tries 5',
+        'set_choose_tries 100',
+        'take default class hdd',
+        'choose indep 4 type host',
+        'chooseleaf indep 8 type osd',
```

Comment:
Use choose rather than chooseleaf here due to https://tracker.ceph.com/issues/62213

Reply:
4 hosts/8 osds seems wrong, should probably be 4 hosts/4 osds, I think?

Reply:
Yes, will changed that